cmd/kueue/main.go

```diff
@@ -72,6 +73,8 @@ import (
 	// +kubebuilder:scaffold:imports
 )
 
+const retryInterval = time.Second * 20
```

Comment:
let's make the variable name specific CRD detection.

In the context of kueue's main `retryInterval` can be confusing.

However, making the retry an option with a default defined in the jobframework might be better.